README.md

```diff
@@ -59,6 +59,88 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Convention ###
+
+With Swift 3, Apple updated the Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) to promote clarity over brevity.
+
+Capitalize all type names with camel case, while function and variable names start with a lower case. Avoid acronyms and abbreviations for clarity and readability.
```

Comment:
This is not easy to read, you talk about type names and functions and variables in the same sentence.. Maybe a bunch of bullet points + examples would help?